File: modules/fbs-core/api/src/main/scala/de/thm/ii/fbs/services/checker/trait/CheckerServiceOnDelete.scala
package de.thm.ii.fbs.services.checker.`trait`

import de.thm.ii.fbs.model.CheckrunnerConfiguration
import de.thm.ii.fbs.model.task.Task

trait CheckerServiceOnDelete {
  def onCheckerConfigurationDelete(task: Task, checkerConfiguration: CheckrunnerConfiguration): Unit
}

File: modules/fbs-core/api/src/main/kotlin/de/thm/ii/fbs/services/v2/checker/trait/CheckerServiceOnDelete.kt
package de.thm.ii.fbs.services.v2.checker.trait

import de.thm.ii.fbs.model.v2.CheckrunnerConfiguration
import de.thm.ii.fbs.model.v2.Task

interface CheckerServiceOnDelete {
    fun onCheckerConfigurationDelete(task: Task, checkerConfiguration: CheckrunnerConfiguration)
}